### PR TITLE
Wait for xtables lock instead of exiting

### PR DESCRIPTION
**Pull request by eyakubovich**

### CODE REVIEW

**Comment by vcaputo** [comment]
lgtm

File: iptables/iptables.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func New() (*IPTables, error) {
 }
 
 // Exists checks if given rulespec in specified table/chain exists
-func (ipt *IPTables) Exists(table, chain string, rulespec...string) (bool, error) {
```
Comment (jonboulle) [inline iptables/iptables.go]: how did this even compile?! (i.e., where are the tests blocking this from getting into master :-)

**Comment by eyakubovich** [comment]
Dup of #3

File: iptables/iptables.go (hunk 1 above)
Comment (eyakubovich) [inline iptables/iptables.go]: fwiw, it is valid Go, just not gofmt'ed.

**Pull request closed without merging**